### PR TITLE
Shallow ensembles

### PR DESCRIPTION
**Pull request by M-R-Schaefer**
WIP just for the visibility / ease of feedback.

### CODE REVIEW

**Comment by M-R-Schaefer** [comment]
This PR has become a bit of a collection of stuff I've done in the last month. Not a particularly good practice, but here we are.

The PR contains the following:
- implementation of shallow ensembles
This includes the model itself as well as two loss functions, NLL and CRPS.
A small refactor of the loss function internals was required to allow them the processing of uncertainties.

- updated use of the jax host callback api (Jaxmd compatible with newest jax version)
- implementation of an ApaxImport Node wich allows the use of standalone trained apax models to be used in IPSuite workflows
